### PR TITLE
Support Python 3.8

### PR DESCRIPTION
**Pull request by tschorr**

### CODE REVIEW

**Comment by tschorr** [comment]
The one failing build for Python 3.7 is a "connection reset by peer" error when buildout tries to download "DocumentTemplate". It looks unrelated, more so since the same test passes in build [#6](https://travis-ci.org/github/plone/Products.isurlinportal/jobs/718914600) later.

**Comment by mauritsvanrees** [comment]
Could you add Python 3.6 as well, while you are at it?
And update tox.ini to include 3.6 and 3.8, for local testing?

Meanwhile, I have added web hooks, so the next PR should have Jenkins integration just like other core packages. 
No Jenkins jobs are needed in this case of course.

Can you add a `news/3.bugfix` snippet though, for clarity?

**Comment by tschorr** [comment]
done.

> Meanwhile, I have added web hooks, so the next PR should have Jenkins integration just like other core packages.

@mauritsvanrees could you add a Jenkins job for Python 3.8? There should be one according to https://github.com/plone/Products.CMFPlone/issues/2890.

File: buildout.cfg [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [buildout]
 extends =
-    https://raw.githubusercontent.com/collective/buildout.plonetest/master/test-5.x.cfg
+    https://raw.githubusercontent.com/collective/buildout.plonetest/master/test-5.2.x.cfg
```
Comment (mauritsvanrees) [inline buildout.cfg:3]: This is not good. In `.travis.yml` we replace `test-5.x` with `test-$PLONE_VERSION.x.cfg`. With your change to 5.2.x, this replacement does not happen anymore, so we only test Plone 5.2 in all Travis jobs.

**Comment by mauritsvanrees** [comment]
>  could you add a Jenkins job for Python 3.8? There should be one according to [plone/Products.CMFPlone#2890](https://github.com/plone/Products.CMFPlone/issues/2890).

No I can't. The jobs are there, and also a PR job for 3.8 is there, but that one can only be manually triggered. There is nothing in the web hooks for this. Could be changed in `mr.roboto` I think.

File: buildout.cfg (hunk 1 above)
Comment (tschorr) [reply buildout.cfg:3]: sorry, this wasn't by intention - rather this is the result of a local tox run (sed is used there to generate the version). I'll revert that.

Comment (tschorr) [reply buildout.cfg]: on second thought, the buildout config created by sed could be named differently to prevent this. But that's maybe for another time.

**Comment by mauritsvanrees** [comment]
Thanks! I will merge.

**Pull request merged**